src/main/editDelete/preProcess.ts

```diff
@@ -0,0 +1,53 @@
+import { Transcription, Word } from 'sharedTypes';
+
+const preProcessTranscript = (
+  jsonTranscript: any,
+  duration: number
+): Transcription => {
+  const numberOfWords: number = jsonTranscript.words.length;
+
+  const processedTranscript: Transcription = {
+    confidence: jsonTranscript.confidence,
+    words: []
+  }
```

Comment:
This is tiny but this object declaration is missing a semicolon. Has linter / prettier been run on this file?

Reply:
It has and there were no errors. Not sure why this wasn't picked up.

Reply:
@LiamTodd

Reply:
My bad!